charts/posit-chronicle/NEWS.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.4.4
+- Add resource requests and limits to the chart.
```

Comment:
Should this be a patch? Are there any situations where this could potentially break something?